app/graphql/types/album_type.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+  class AlbumType < BaseObject
+    field :id, ID, null: false
+    field :title, String, null: false
+    field :artist, String, null: false
```

Comment:
field artist in the API comes as userId so this field won't work.

Reply:
Done!